Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -181,4 +181,155 @@ namespace MongoDB.Driver.Examples
 }
 ```
 
-**Coming soon:** An example using the community version and demonstrating explicit encryption/decryption.
+### Explicit Encryption and Decryption
```

Comment:
Not sure how many # should be here. Thoughts?

Reply:
Presumably this new example is ONE example in the following section:

    ## Examples

In which case we should also add a heading (with ###) before line 43. Something like:

    ### Automatic client-side encryption